company-profiles/15five.md

```diff
@@ -18,7 +18,7 @@ Over half of company works remote. We communicate via [slack](https://www.slack.
 
 ## Region
 
-North America and Europe. 
+The Americas and Europe. 
```

Comment:
are you using `The Americas` to mean including `South America` as well? but from where do you see ? Could you give a link of information source?
I've just checked their open list, I only see the US based remote in `The Americas` region, not even including Canada,

- https://www.15five.com/careers/
- https://jobs.lever.co/15five